src/main/java/com/example/gradecalculator/model/UserEditTO.java

```diff
@@ -4,15 +4,30 @@
 import lombok.Getter;
 import lombok.Setter;
 
-
+@Getter
+@Setter
 public class UserEditTO {
-    @Getter
-    @Setter
+
     @NotBlank
     private String firstName;
 
-    @Getter
-    @Setter
     @NotBlank
     private String lastName;
-}
\ No newline at end of file
+
+    // Ensure getters and setters or Lombok works properly
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
```

Comment:
class is annotated with setter and getter attribute, why do you need those methods?